### PR TITLE
Updating data attribute in MessageReceived

### PR DESCRIPTION
**Pull request by sarojkpr**
image

### CODE REVIEW

**Review by elopezanaya — APPROVED** [review]

File: chat-widget/src/plugins/newMessageEventHandler.ts [hunk 1]
```diff
@@ -20,15 +20,13 @@ export const createOnNewAdapterActivityHandler = (chatId: string, userId: string
     const raiseMessageEvent = (activity: IActivity) => {
         if (activity?.type === Constants.message) {
             const payload = {
+                // To identify hidden contents vs no content
                 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                text: (activity as any)?.text,
-                id: activity?.id,
+                text: (activity as any)?.text?.length >= 1 ? "*contents hidden*" : "",
```
Comment (charliewang95) [inline chat-widget/src/plugins/newMessageEventHandler.ts:25]: Why >= 1 not >1?

Comment (charliewang95) [inline chat-widget/src/plugins/newMessageEventHandler.ts:25]: Can we add length of text as well?” Content hidden (20 chars)”

Comment (sarojkpr) [reply chat-widget/src/plugins/newMessageEventHandler.ts]: image

Comment (sarojkpr) [reply chat-widget/src/plugins/newMessageEventHandler.ts]: This is for single char message validation

**Pull request merged**